go.mod

```diff
@@ -218,6 +218,7 @@ require (
 )
 
 replace (
+	cosmossdk.io/math => cosmossdk.io/math v1.4.0
```

Comment:
It's already 1.4.0, if you don't want to prevent future updates, i think it's not necessarily to set a replacement

Reply:
Doesn't this replace forces for v1.4.0 to be used in all dependencies?